plugins/modules/zos_find.py

```diff
@@ -0,0 +1,791 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) IBM Corporation 2020
+# Apache License, Version 2.0 (see https://opensource.org/licenses/Apache-2.0)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['stableinterface'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r"""
+---
+module: zos_find
+version_added: "2.9"
+short_description: Find matching data sets
+description:
+  - Return a list of data sets based on specific criteria.
+  - Multiple criteria can be added (AND'd) together.
+  - Use the M(find) module to find USS files.
```

Comment:
The only drawback to using L(...,...) is command line ansible-doc sort of looks off with a URL but i still think that is fine, opinion from @RadhaV2020 ?

```
> ZOS_FIND    (/Users/ddimatos/.ansible/collections/ansible_collections/ibm/ibm_zos_core/plugins/modules/zos_find.py)

        Return a list of data sets based on specific criteria. Multiple criteria can be added (AND'd) together. Use the
        L(find,https://docs.ansible.com/ansible/latest/modules/find_module.html) module to find USS files.
```

Reply:
Let's not use a link inline. If we must add a link, include it in a separate, generic statement, may be in the example section.

Reply:
@RadhaV2020 im not sure how this would be different in the example section , maybe an example , ￼￼￼￼examples are just snippets of code, no doc there. I'm ok with this even as ansible doc , anyone using ansible doc is technical enough to follow a url , also most modern terminals open links . I'll wait to hear from you , should not hold up a pull request , easy to change it later before we release if more time is needed

Reply:
@ddimatos sorry for the delay on this one. Had a brief chat with @asifmahmud on this one. My point was towards making the capabilities of find and zos_find explicit so it is clear to the reader that about what is possible with zos_find and what isn't.